x-pack/plugins/cloud_security_posture/public/pages/findings/findings_by_resource_table.test.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import * as TEST_SUBJECTS from './test_subjects';
+import { FindingsByResourceTable } from './findings_by_resource_table';
+import * as TEXT from './translations';
+import type { PropsOf } from '@elastic/eui';
+import Chance from 'chance';
+
+const chance = new Chance();
+
+const getFakeFindingsByResource = () => ({
+  resource_id: chance.guid(),
+  cluster_id: chance.guid(),
+  cis_section: chance.word(),
+  failed_findings: {
+    total: chance.integer(),
+    normalized: chance.integer({ min: 0, max: 1 }),
```

Comment:
This will be either `0` or `1`, yea?

Reply:
woops 😄 

yeah, intention was a value between 0 and 1. i'll fix this. 

worth noting though this test suite doesn't check for `failed_findings`.  comparing the values with `html` markup is slightly harder than just texts so i skipped it 😅 . it tests all other columns tho.

do you think we should add that case in this PR ?